.about.yml

```diff
@@ -0,0 +1,105 @@
+---
+# .about.yml project metadata
+#
+# Short name that acts as the project identifier (required)
+name: project-dashboard
+
+# Full proper name of the project (required)
+full_name: '18F Project Dashboard'
+
+# The type of content in the repo
+# values: app, docs, policy
+type: app
+
+# Describes whether a project team, working group/guild, etc. owns the repo (required)
+# values: guild, working-group, project
+owner_type: working-group
+
+# Name of the main project repo if this is a sub-repo; name of the working group/guild repo if this is a working group/guild subproject
+parent:
+
+# Maturity stage of the project (required)
+# values: discovery, alpha, beta, live
+stage: alpha
+
+# Whether or not the project is actively maintained (required)
+# values: active, deprecated
+status: active
+
+# Description of the project
+description: >
+  A dashboard based on the Dashing framework that shows info and statistics
+  on all 18F GitHub repos, such as whether CI builds are passing or failing,
+  aggregate amount of pull requests and issues, test coverage percentages,
+  and code quality metrics.
+
+# Should be 'true' if the project has a continuous build (required)
+# values: true, false
+testable: true
+
+# Team members contributing to the project (required)
+# Items:
+# - github: GitHub user name
+#   id: Internal team identifier/user name
+#   role: Team member's role; leads should be designated as 'lead'
+team:
+- github: monfresh
+  id: Moncef
+  role: developer
+
+# Partners for whom the project is developed
+partners:
+-
+
+# Brief descriptions of significant project developments
+milestones:
+-
+
+# Technologies used to build the project
+stack:
+- Ruby
+- Sinatra
+- Dashing
+
+# Brief description of the project's outcomes
+impact:
+
+# Services used to supply project status information
+# Items:
+# - name: Name of the service
+#   category: Type of the service
+#   url: URL for detailed information
+#   badge: URL for the status badge
+services:
+- name: GitHub
+  category: API
+  url: https://developer.github.com/
+- name: Travis
+  category: API
+  url: http://docs.travis-ci.com/api/
+- name: '18F Projects'
+  category: API
+  url: https://team-api.18f.gov/public/api/projects/
+
+# Licenses that apply to the project and/or its components (required)
+# Items by property name pattern:
+#   .*:
+#     name: Name of the license from the Software Package Data Exchange (SPDX): https://spdx.org/licenses/
+#     url: URL for the text of the license
+licenses:
+  placeholder_label:
+
+# Blogs or websites associated with project development
+blog:
+-
+
+# Links to project artifacts
+# Items:
+# - url: URL for the link
+#   text: Anchor text for the link
+links:
```

Comment:
These should be in the form described in the comment, as objects with `url:` and `text:` properties.